common/scala/src/main/scala/whisk/http/BasicHttpService.scala

```diff
@@ -165,7 +165,7 @@ object BasicHttpService {
   /**
    * Starts an HTTPS route handler on given port and registers a shutdown hook.
    */
-  def startHttpsService(route: Route, port: Int, config: WhiskConfig)(implicit actorSystem: ActorSystem,
+  def startHttpsService(route: Route, port: Int, config: HttpsConfig)(implicit actorSystem: ActorSystem,
```

Comment:
This method is pretty similar to `startHttpService`, should we perhaps merge them to something like:

```
  def startHttpService(route: Route, port: Int, config: Option[HttpsConfig] = None)(
    implicit actorSystem: ActorSystem,
    materializer: ActorMaterializer): Unit = {

    implicit val executionContext = actorSystem.dispatcher
    val context = config.map(c => Https.connectionContext(c)).getOrElse(Http().defaultServerHttpContext)
    Http().bindAndHandle(route, "0.0.0.0", port, connectionContext = context)
  }
```